.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x]
```

Comment:
No point running tests on Node 12 since we no longer have any Node 12 apps.

I did decide to leave the matrix, so we can easily add more (such as Node 16) when the time comes.